tools/cloud-build/provision/list_tests.py

```diff
@@ -63,4 +76,8 @@ def list_builds(start_time=30, end_time=5*60):
                         help='minutes since midnight to run the last test, e.g. 300 = 05:00')
     args = parser.parse_args()
 
-    print(json.dumps(list_builds(args.start_time, args.end_time)))
+    assert args.start_time < args.end_time
+    builds = list_builds()
+    schedule = schedule_evenly(builds, args.start_time, args.end_time)
+    #schedule = schedule_consistently(builds, args.start_time, args.end_time)
```

Comment:
I think my preference is for evenly. If we hit a bad sequence we can always use the hash to shake it up. With consistent scheduling we get random timing and much more likely to get contention. I don't know if consistency benefit makes up for this. Feel free to leave it in unused in case we want it later.